jumping_quadrupeds/smaq/agent.py

```diff
@@ -60,12 +60,20 @@ def __init__(
         # models
         self.model = model
         self.model_ema = model_ema
-        self.actor = Actor(self.model.out_dim, action_space, feature_dim, hidden_dim, log_std_init, use_actor_ln).to(
-            device
-        )
+        if action_space.__class__.__name__ == "Discrete":
+            self.discrete_actions = True
+            action_dim = action_space.n
+            self.actor = DiscreteActor(self.model.out_dim, action_space, feature_dim, hidden_dim, log_std_init,
+                                       use_actor_ln).to(device)
+
+        else:
+            self.discrete_actions = False
+            action_dim = action_space.shape[0]
+            self.actor = Actor(self.model.out_dim, action_space, feature_dim, hidden_dim, log_std_init,
+                               use_actor_ln).to(device)
```

Comment:
Great - once we copy this change to DRQ-V2 and PPO, should work with the new Discrete Critic